tests/integration/cartography/intel/aws/test_s3.py

```diff
@@ -48,6 +48,35 @@ def test_load_s3_buckets(neo4j_session, *args):
     assert actual_nodes == expected_nodes
 
 
+def test_load_s3_buckets_sync_metadata(neo4j_session, *args):
```

Comment:
Nit: add comments in the test for 

```
# Arrange

# Act

# Assert
```

to help even more with readability

Reply:
Added.